service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/CbsSenderLinkHandler.java

```diff
@@ -108,41 +108,38 @@ public int sendAuthenticationMessage(UUID authenticationMessageCorrelationId)
         properties.setReplyTo(CBS_REPLY);
         outgoingMessage.setProperties(properties);
 
-        Map<String, Object> userProperties = new HashMap<>();
```

Comment:
This object is just renamed to "applicationProperties" to better reflect what it contains